CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Changelog
 
 ## In Development
-
+* Update `rabbitmq-ha` 3rd party chart from `1.44.1` to `1.46.1` (#158) (by @moonrail)
+* Disable newly introduced `rabbitmq-ha` prometheus operator by default (#158) (by @moonrail)
```

Comment:
This could be omitted as there's too verbose changelog for a single PR and we stick with the previous default.
```suggestion
```

Reply:
This line is now removed - wasn't sure about three lines of Changelog for one PR either when writing it